setup.py

```diff
@@ -146,10 +145,10 @@
     'invenio-records-ui>=1.0.0',
     'invenio-records>=1.0.0',
```

Comment:
Minimum versions of packages should also be bumped (if they were bumped in `requirements.txt`)